src/components/PopupWithForm.js

```diff
@@ -22,6 +23,14 @@ export default class PopupWithForm extends Popup {
     return inputsItems;
   }
 
+  buttonLoading(loading) {
```

Comment:
The name of the function should begin with a verb, reflecting the action that the function performs. Next, the elements to which the action refers are described. `renderLoading` would be good.

Reply:
Corrected!